ChangeLog.md

```diff
@@ -30,6 +30,8 @@
   * Add personal access token management (Nathan Walters).
 
   * Add `allow-complex` attribute for `pl-symbolic-input` (Tim Bretl).
+  
+  * Add read-only API for instructors to access assessment data (Nathan Walters).
```

Comment:
This might be a dumb question, but do we have any tests for the api?

Reply:
No tests at present :/ I would like to add them later though.